ReactiveCocoa/Swift/UnidirectionalBinding.swift

```diff
@@ -0,0 +1,315 @@
+import enum Result.NoError
+
+infix operator <~ {
+	associativity right
+
+	// Binds tighter than assignment but looser than everything else
+	precedence 93
+}
+
+public protocol BindingConsumer: class {
```

Comment:
@mdiep The trouble with a generic `Bindable` name is that it's easily confused with two-way "bindings". I don't love the `BindingConsumer` name despite using it myself, and perhaps `Consumer` would be more apt.

`ValueConsumer` could work as well, however the `<~` operator really sends events.